features/step_definitions/auction_create_steps.rb

```diff
@@ -11,7 +11,7 @@
 end
 
 Given(/^there is a closed auction$/) do
-  @auction = FactoryGirl.create(:auction, :closed, :with_bids)
+  @auction = FactoryGirl.create(:auction, :closed, :with_bids, :c2_budget_approved)
```

Comment:
it does feel kind of wrong that a generally closed auction must be `c2_budget_approved` -- wonder if we should have a special step for a "default pcard closed auction" auction